documentation/dev_running_from_source.markdown

```diff
@@ -159,6 +159,25 @@ run the following:
 
     $ lein run -c /path/to/puppetserver.conf
 
+Running the Agent
+-----
+
+Use a command like the one below to run an agent against your running puppetserver:
+
+        puppet agent --confdir ~/.puppetlabs/etc/puppet \
```

Comment:
@camlow325 just tested it out locally, you're right about the server getting pulled from the confdir and server no longer being necessary. I'll remove that part